hoomd/md/CMakeLists.txt

```diff
@@ -50,6 +50,8 @@ set(_md_sources module-md.cc
 
 set(_md_headers ActiveForceComputeGPU.h
                 ActiveForceCompute.h
+		ActiveForceConstraintCompute.h
+		ActiveForceConstraintComputeGPU.h
```

Comment:
I think you may want to add a few more of your files to the various sources.

Reply:
Which ones did I miss?